core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -55,4 +55,16 @@ private MetadataColumns() {
   public static Set<Integer> metadataFieldIds() {
     return META_IDS;
   }
+
+  public static NestedField get(String name) {
+    return META_COLUMNS.get(name);
+  }
+
+  public static boolean isMetadataColumn(String name) {
+    return META_COLUMNS.containsKey(name);
+  }
+
+  public static boolean nonMetadataColumn(String name) {
```

Comment:
nit: should it be `isNonMetadataColumn` to indicate it is a boolean flag?

Reply:
Yeah, we could do that. I was following the slightly shorter Scala convention, where there are methods like `nonEmpty`. I prefer this way, but if others agree I'm happy to change it.

Reply:
Let's leave it, I don't mind as long as it wasn't an oversight and there is an idea/convention being followed.